lib/patterns/calculation.rb

```diff
@@ -0,0 +1,45 @@
+module Patterns
+  class Calculation
+    require 'active_support/cache'
+    CACHE_ENGINE = defined?(Rails) && Rails.respond_to?(:cache) ? Rails.cache : ActiveSupport::Cache::MemoryStore.new
+
+    class << self
+      attr_accessor :cache_expiry_every
```

Comment:
Yes, it will - it's an accessor on an instance of a class, so it's actually referring to the `class instance variable`, not to the `class variable`, and as so it should work as expected with inheritance (i.e. it will be possible to change the value of `cache_expiry_every` on the derived class without actually affecting the original class, as opposed to class variables).
But assuming we have `active_support` there, it's probably still better to use `class_attribute` anyway :)